assets/apps/components/src/Controls/ColorControl.js

```diff
@@ -86,20 +102,64 @@ const ColorControl = ({
 					<>
 						{/* eslint-disable-next-line  jsx-a11y/anchor-has-content */}
 						<a href="#color-picker" />
+						{allowGradient && (
+							<ButtonGroup
+								className="color-toggle"
+								style={{ marginBottom: '8px' }}
+							>
+								<Button
+									isPrimary={activePanel === 'color'}
+									isSecondary={activePanel !== 'color'}
+									onClick={() => setActivePanel('color')}
+								>
+									{__('Color', 'neve')}
+								</Button>
+								<Button
+									isPrimary={activePanel === 'gradient'}
+									isSecondary={activePanel !== 'gradient'}
+									onClick={() => setActivePanel('gradient')}
+								>
+									{__('Gradient')}
```

Comment:
Missing text-domain for translation

Reply:
It uses the default WordPress translation, is this not allowed? `Gradient` is already translated inside WP.

Reply:
It won't know to translate a string from the theme based on WordPress' translation file. That's why we have other simple strings like "Button" translated into the theme.